knife-ec2.gemspec

```diff
@@ -17,7 +17,7 @@ Gem::Specification.new do |s|
   s.executables  = `git ls-files -- bin/*`.split("\n").map { |f| File.basename(f) }
   s.required_ruby_version = ">= 2.2.2"
 
-  s.add_dependency 'fog-aws',       '~> 0.7'
+  s.add_dependency 'fog-aws',       '~> 1.2'
```

Comment:
Do we absolutely need 1.2, or could we be okay with `'~> 1.0'` ? 1.2 was release a bit ago, January 20, 2017, and in general it's good to be cautious about bumping deps to the edge because they can bump other deps to the end and bring about gem version conflicts when we try to build chef-dk.